src/features/courses/course-route.ts

```diff
@@ -371,12 +371,19 @@ router.post('/topic/:topicId/startExport',
         // If we're just checking, return what's already here.
         if (query.force === false) {
             next(httpResponse.Ok('Details', exportDetails));
-            // If we should start a new export
-        } else if (helper.shouldStartNewExport(topic)) {
-            helper.start({topic, professorUUID: professor.uuid}).then(() => logger.info('Finished uploading.'));
-            next(httpResponse.Ok('Loading', exportDetails));
         } else {
-            next(httpResponse.Ok('Already exported', exportDetails));
+            try {
+                helper.start({
```

Comment:
don't know if this can throw an exception but should probably have a catch to avoid unhandled promise rejection